.gitignore

```diff
@@ -10,6 +10,7 @@
 # Ignore all logfiles and tempfiles.
 /log/*
 /tmp/*
+/spec/examples.txt
```

Comment:
I added this bellow, but probably in the wrong place. We can remove the dupe later.